sdk/storage/azblob/sas/service.go

```diff
@@ -16,28 +16,29 @@ import (
 )
 
 // BlobSignatureValues is used to generate a Shared Access Signature (SAS) for an Azure Storage container or blob.
-// For more information, see https://docs.microsoft.com/rest/api/storageservices/constructing-a-service-sas
+// For more information on creating service sas, see https://docs.microsoft.com/rest/api/storageservices/constructing-a-service-sas
+// For more information on creating user delegation sas, see https://docs.microsoft.com/rest/api/storageservices/create-user-delegation-sas
 type BlobSignatureValues struct {
-	Version                    string    `param:"sv"`  // If not specified, this defaults to Version
-	Protocol                   Protocol  `param:"spr"` // See the Protocol* constants
-	StartTime                  time.Time `param:"st"`  // Not specified if IsZero
-	ExpiryTime                 time.Time `param:"se"`  // Not specified if IsZero
-	SnapshotTime               time.Time
-	Permissions                string  `param:"sp"` // Create by initializing a ContainerSASPermissions or BlobSASPermissions and then call String()
-	IPRange                    IPRange `param:"sip"`
-	Identifier                 string  `param:"si"`
-	ContainerName              string
-	BlobName                   string // Use "" to create a Container SAS
-	Directory                  string // Not nil for a directory SAS (ie sr=d)
-	CacheControl               string // rscc
-	ContentDisposition         string // rscd
-	ContentEncoding            string // rsce
-	ContentLanguage            string // rscl
-	ContentType                string // rsct
-	BlobVersion                string // sr=bv
-	PreauthorizedAgentObjectId string
-	AgentObjectId              string
-	CorrelationId              string
+	Version              string    `param:"sv"`  // If not specified, this defaults to Version
+	Protocol             Protocol  `param:"spr"` // See the Protocol* constants
+	StartTime            time.Time `param:"st"`  // Not specified if IsZero
+	ExpiryTime           time.Time `param:"se"`  // Not specified if IsZero
+	SnapshotTime         time.Time
+	Permissions          string  `param:"sp"` // Create by initializing a ContainerSASPermissions or BlobSASPermissions and then call String()
+	IPRange              IPRange `param:"sip"`
+	Identifier           string  `param:"si"`
+	ContainerName        string
+	BlobName             string // Use "" to create a Container SAS
+	Directory            string // Not nil for a directory SAS (ie sr=d)
+	CacheControl         string // rscc
+	ContentDisposition   string // rscd
+	ContentEncoding      string // rsce
+	ContentLanguage      string // rscl
+	ContentType          string // rsct
+	BlobVersion          string // sr=bv
+	AuthorizedObjectId   string // saoid
```

Comment:
`Id` should be all caps (see also the next line).

Reply:
Done